charts/dev/theme-park/Chart.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v2
+appVersion: "1.9.4"
+dependencies:
+- name: common
+  repository: https://library-charts.truecharts.org
+  version: 9.3.19
+deprecated: false
+description: Custom themes for your favorite apps!
+home: https://github.com/truecharts/apps/tree/master/charts/dev/theme.park
+icon: https://github.com/GilbN/tp-docs/raw/main/docs/site_assets/logo.png
```

Comment:
Add the icon on truecharts/pub 
and use that link instead (simillar to the other apps)